docs/intro_tutorial.rst

```diff
@@ -436,18 +406,12 @@ partial transpose over the first subsystem yields the following matrix
                     7 & 8 & 15 & 16
                 \end{pmatrix}.
   
-.. code-block:: python
+.. jupyter-execute::
 
-    >>> from toqito.channels import partial_transpose
-    >>> import numpy as np
-    >>> test_input_mat = np.array(
-    ...     [[1, 2, 3, 4], [5, 6, 7, 8], [9, 10, 11, 12], [13, 14, 15, 16]]
-    ... )
-    >>> partial_transpose(test_input_mat, [0])
-    array([[ 1,  2,  9, 10],
-           [ 5,  6, 13, 14],
-           [ 3,  4, 11, 12],
-           [ 7,  8, 15, 16]])
+     from toqito.channels import partial_transpose
+     import numpy as np
+     test_input_mat = np.arange(1, 17).reshape(4, 4)
+     partial_transpose(test_input_mat, [0])
 
```

Comment:
Same as my other comment, there were a couple of changes made to this tutorial in other PRs. Those code blocks are still in the old format. 

I'll leave it up to you. If you would prefer I create a separate issue for someone else to handle it, I am fine with it.